hera_qm/xrfi.py

```diff
@@ -196,7 +188,7 @@ def xrfi(d, f=None, K=8, sig_init=6, sig_adj=2):
     Returns:
         bool array: array of flags
     """
-    nsig = detrend_medfilt(d, K=K)
+    nsig = detrend_medfilt(d, Kt=Kt, Kf=Kf)
     f = watershed_flag(np.abs(nsig), f=f, sig_init=sig_init, sig_adj=sig_adj)
     return f
 
```

Comment:
I'm not sure, there's no other reference to it besides this as far as I'm aware.